### PR TITLE
Add Strega Devices

### PR DESCRIPTION
**Pull request by i3mSuperMan**
#### Summary

Added Strega devices as per the customer
- smart-valve

#### Notes for Reviewers

As per request from the customer, we changed the `id` and `name` in the `index.yaml` from `strega-sprl` to `strega`.

### CODE REVIEW

**Review by Jaime-Trinidad — COMMENTED** [review]
vendor changes ok, index add device ok, two profiles ok, device information with profile id and codec ok, js file for decode uplink, encode and decode downlink ok, codec.yalm looks ok

File: vendor/strega/smart-valve-codec.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# Uplink decoder decodes binary data uplink into a JSON object (optional)
+# For documentation on writing encoders and decoders, see: https://thethingsstack.io/integrations/payload-formatters/javascript/
+uplinkDecoder:
+  fileName: smart-valve.js
+  # Examples (optional)
```
Comment (johanstokking) [inline vendor/strega/smart-valve-codec.yaml:5]: Will you provide examples? Do you need help with that?

**Review by johanstokking — COMMENTED** [review]
I'm seeing a few `._*` files checked in, like `vendor/strega/._smart-valve.jpg`. Please remove those

File: vendor/strega/smart-valve-codec.yaml (hunk 1 above)
Comment (surendar94) [reply vendor/strega/smart-valve-codec.yaml:5]: When i give the example the code is giving an error.

My input is as follows: "3330313131" and port is 4

The output should be 
```
{
  "Battery": 63,
  "Cable": "0",
  "Class": "0",
  "DI_0": "0",
  "DI_1": "0",
  "Fraud": "0",
  "Leakage": "0",
  "Port": 4,
  "Power": "0",
  "Process": "true",
  "Status": "00000001",
  "Tamper": "0",
  "Valve": "1"
}
```

when i check direct in TTN or TTI i am getting the exact output, but not here.

Can you help me how to make it?

Comment (johanstokking) [reply vendor/strega/smart-valve-codec.yaml:5]: You need to pass the input bytes as array. If the input you gave is hex, you should pass it like `[0x33, 0x30, 0x31, 0x31, 0x31]`

Comment (surendar94) [reply vendor/strega/smart-valve-codec.yaml:5]: Can you please review the latest file. and approve it. We have not added example to it. If required we will add it later.

Comment (surendar94) [reply vendor/strega/smart-valve-codec.yaml:5]: Dear Johan. Any updates for the latest and corrected file?

**Review by Jaime-Trinidad — APPROVED** [review]
The files look complete except the example in smart-valve.codec.yalm that they will provide later.

File: vendor/strega/smart-valve-codec.yaml (hunk 1 above)
Comment (Jaime-Trinidad) [reply vendor/strega/smart-valve-codec.yaml:5]: Hi, I review files and they look fine, we only need last aproval, please don´t forget to add the example and if you need any support on this please tell me.

**Comment by surendar94** [comment]
> The files look complete except the example in smart-valve.codec.yalm that they will provide later.

Thank you for your review. How can i merge it?

File: vendor/strega/smart-valve-codec.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  #examples:
+  # - description: Periodic Uplink
+  #  input:
+  #   fPort: 4
+  #  bytes: [333138383223633D77DB]
```
Comment (johanstokking) [inline vendor/strega/smart-valve-codec.yaml:10]: @surendar94 what I mean is:

```suggestion
  #  bytes: [0x33, 0x31, 0x38, 0x38, 0x32, 0x23, 0x63, 0x3D, 0x77, 0xDB]
```

Comment (ymgupta) [reply vendor/strega/smart-valve-codec.yaml:10]: @johanstokking, Could you merge this PR? Surendar says that they will add example if required later.

**Comment by johanstokking** [comment]
I'm fine with that too, up to Strega. Examples are very useful and become increasingly useful as we add functionality around it.

In any case, CI is failing on:

```
./vendor/strega/smart-valve.jpg extension is incorrect, it should be png
```

**Comment by i3mSuperMan** [comment]
@johanstokking,

It seems the device didn’t get reflected yet on the Console. May we know if there is an issue? Could you let us know the time before it gets reflected?

**Comment by cBashTN** [comment]
@i3mSuperMan In your profiles you link to ```vendorProfileID: 42```.  But you are 401. Maybe this is the problem?

**Comment by johanstokking** [comment]
The device repository gets updated only on releases only. We track an internal issue to refresh automatically but as we are on a 2 week release cadence, we didn't prioritize auto refresh so far.

**Pull request merged**